### PR TITLE
docs: adds GeoJSON import documentation [DHIS2-7171]

### PR DESCRIPTION
**Pull request by jbee**
Documentation for changes in new GeoJSON features from:
* https://github.com/dhis2/dhis2-core/pull/10838 
* https://github.com/dhis2/dhis2-core/pull/10863
* https://github.com/dhis2/dhis2-core/pull/10875
* https://github.com/dhis2/dhis2-core/pull/10963

### CODE REVIEW

File: src/developer/web-api/metadata.md [hunk 1]
```diff
@@ -1071,6 +1072,7 @@ ignored:
 }
 ```
 
+!INCLUDE "geo-json.md"
```
Comment (jbee) [inline src/developer/web-api/metadata.md:1075]: @Philip-Larsen-Donnelly I think this is the first time someone actually uses includes. Maybe you can check if this actually works? Thanks!

Comment (Philip-Larsen-Donnelly) [reply src/developer/web-api/metadata.md:1075]: Includes should still work. I can test this. 
We have generally moved away from includes as it breaks the concept of providing an edit button for each page in the documentation portal. (The edit button will take the user to the file in the docs git repo - but will not follow includes).

Comment (jbee) [reply src/developer/web-api/metadata.md:1075]: That is an unfortunate complication. I thought these files get so large includes would help to break this up and make the edit in an IDE manageable. So at the same time this breaks the portals edit.

Comment (Philip-Larsen-Donnelly) [reply src/developer/web-api/metadata.md:1075]: Yeah, the original idea of the includes was to make it more manageable, but I feel that the accessibility from the portal is pretty important; and we can consider breaking sections into multiple pages if they become too large and unmanageable.

Comment (jbee) [reply src/developer/web-api/metadata.md:1075]: Yes, I think breaking the metadata.md in multiple pages is the best. That way the imported one can just become one of the pages in the sequence of pages.

**Review by larshelge — APPROVED** [review]

**Pull request merged**